e2e/node/acp.test.ts

```diff
@@ -129,12 +124,8 @@ describe("An ACP Solid server", () => {
     );
   });
 
-  it("advertises its ACLs as ACP AccessControlResources", async () => {
-    expect(await acp.isAcpControlled(sessionResource, fetchOptions)).toBe(true);
-  });
```

Comment:
Is this something that doesn't have an equivalent in the latest API?

Reply:
Indeed it does; added back in in https://github.com/inrupt/solid-client-js/pull/2294/commits/60d976a2ca5598c0b2b1772557a6ac49ffbb4d9b.